website/src/routes/apiv2.js

```diff
@@ -0,0 +1,776 @@
+/*
```

Comment:
Does this file require anything that uses `coffee-script`? If so, we may need to do `require('coffee-script`);` so that it will work correctly.